Infrastructure/workstations.json

```diff
@@ -1,12 +1,15 @@
 [
-    {"Host":"vega", "IPAddress":"67.69.105.136", "Status":"unknown"},
-    {"Host":"rob", "IPAddress":"67.69.105.137", "Status":"unknown"},
-    {"Host":"ps", "IPAddress":"67.69.105.138", "Status":"unknown"},
-    {"Host":"italy", "IPAddress":"67.69.105.142", "Status":"unknown"},
-    {"Host":"canada", "IPAddress":"67.69.105.143", "Status":"unknown"},
-    {"Host":"narnia", "IPAddress":"67.69.105.159", "Status":"unknown"},
-    {"Host":"spain", "IPAddress":"67.69.105.166", "Status":"unknown"},
-    {"Host":"rome", "IPAddress":"67.69.105.170", "Status":"unknown"},
-    {"Host":"switzerland", "IPAddress":"67.69.105.132", "Status":"unknown"},
-    {"Host":"japan", "IPAddress":"67.69.105.139", "Status":"unknown"}
+    {"Host":"bbb", "IPAddress":"bbb.cdot.systems", "Status":"unknown"},
+    {"Host":"vega", "IPAddress":"vega.cdot.systems", "Status":"unknown"},
+    {"Host":"rob", "IPAddress":"rob.cdot.systems", "Status":"unknown"}, 
+    {"Host":"ps", "IPAddress":"ps.cdot.systems", "Status":"unknown"},
+    {"Host":"italy", "IPAddress":"italy.cdot.systems", "Status":"unknown"},
+    {"Host":"canada", "IPAddress":"canada.cdot.systems", "Status":"unknown"},
+    {"Host":"narnia", "IPAddress":"narnia.cdot.systems", "Status":"unknown"},
+    {"Host":"ireland", "IPAddress":"ireland.cdot.systems", "Status":"unknown"},
```

Comment:
Can you remove `ireland` (it's Yan's machine)?

Reply:
Removed